ui/components/app/transaction-detail-item/index.scss

```diff
@@ -22,9 +29,10 @@
     }
   }
 
-  &__total {
+  & h6.transaction-detail-item__total {
     font-weight: bold;
     color: $ui-black;
```

Comment:
Hi, I understand that the `font-weight` and `color` might be pre-existing but aren't they already covered in <Typography/> ?

Reply:
yep, it was not noticed as values overriding them were similar.